Automattic/wp-calypso
# Premium Content: Remove from the Editing Toolkit plugin

#### Changes proposed in this Pull Request

* Remove the Premium Content block from editing toolkit as it is being added to Jetpack.

#### Testing instructions



* Load this branch
* Open the editor and confirm that you cannot see the Premium Content block
* Load the master branch of Jetpack, and confirm that the Premium Content block continues to work (you will need to load it on a WordPress.com sandboxed site).
